lib/order.rb

```diff
@@ -0,0 +1,14 @@
+class Order
```

Comment:
Perfect class for the customer to access and push their order to. Does what it says on the tin. Good use of string interpolation and shortcut methods